pages/upload.tsx

```diff
@@ -71,7 +71,7 @@ function Upload() {
 
     const { reference } = currentSchema
     const defaultState = {
-      contacts: { uploader: user.id },
+      contacts: { uploader: [{ kind: 'user', id: user.id }] },
```

Comment:
Could use an enum for `kind` so that there's less chance of typos. Would be nice if we could use the same enum in our mongoose schema definition, so that we could manage both in one place.

Would something like this work?
```
export enum Kind {
  USER = 'user',
}

// example usage

const defaultState = contacts: { uploader: [{ kind: 'user', id: user.id }] },

// schema definition

kind: {
  type: String,
  enum: Object.values(Kind),
  required: true,
  },
```